study.md

```diff
@@ -67,5 +67,10 @@ Starting with a client making a GET request to a particular URL, describe how
 the parts of Rails interact to produce and send a response.
 
 ```md
-<!-- your answer here -->
+Client requests 'GET /people/1'
+Server recieves 'GET /people/1' and gets the correct controlled in motion 'people'
```

Comment:
The router receives the request and, if the route is valid, determines which controller action to use.